### DISCUSSION TITLE
A project template

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#2780] [Ideas / Feature Requests]
I found a template which is very beautiful and powerfull,It's alsl based on fastapi and vue3,If nice gui could make such template.
Here is the [demo ](https://kinit.ktianc.top/login) (user:15020221010 password:kinit2022) and the [repo](https://github.com/vvandk/kinit)

**Comment by falkoschindler** [comment]
Hi @v479038280,

Can you, please, elaborate a bit? What kind of "template" is it? What is its purpose? How would you use it?

**Reply by v479038280** [reply]
a template for a project,with navgation items
![微信截图_20240330223901](https://github.com/zauberzeug/nicegui/assets/25096045/274bd602-6edd-4ba9-8008-660fb4e632b9)
,also such page tab   is very convient  
![微信截图_20240330224139](https://github.com/zauberzeug/nicegui/assets/25096045/137674cc-779d-42d6-a499-deaf7db9080f)
.  and user mangerment etc
